src/registrar/tests/test_feature_flags.py

```diff
@@ -0,0 +1,167 @@
+from waffle.decorators import flag_is_active
+from django.test import TestCase, Client, RequestFactory
+from registrar.models import (
+    WaffleFlag,
+    User,
+    Contact,
+    UserGroup,
+)
+from registrar.tests.common import create_superuser, create_staffuser, create_user
+
+
+class TestFeatureFlags(TestCase):
```

Comment:
Is this just testing the functionality of the waffle library? We don't typically test 3rd libraries, otherwise we would be testing things the the zipper library etc.

Reply:
To me this just sounds like testing that we can use the 3rd party libarary per it's documentation, which is already tested in their repo. Not very different than tested a sum function gets the right value no matter what we change. Maybe point me to a particular test here that you're seeing as beneficial to keep?